src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx

```diff
@@ -540,6 +542,10 @@ int main_(int argc, const char** argv) {
   for (auto psKernel : addPsKernels) 
     xclBin.addPsKernel(psKernel);
   
+  // -- Add Fixed Kernels files
+  for (auto kernel : addKernels) 
```

Comment:
const auto& ?

Reply:
Ugh!  Missed code reviewing for this in this file.   I have corrected this issue along with the other ones that I found in this file.